bundle/compliance/policy/gcp/compute/ensure_no_use_of_default_sa.rego

```diff
@@ -0,0 +1,31 @@
+package compliance.policy.gcp.compute.ensure_default_sa
+
+import data.compliance.policy.gcp.data_adapter
+import future.keywords.every
+import future.keywords.if
+
+is_default_sa(sa) if {
+	endswith(sa.email, "-compute@developer.gserviceaccount.com")
+}
+
+is_gke_instance(instance) if {
+	startswith(instance.name, "gke-")
+}
+
+is_default_sa_with_access(sa) if {
+	is_default_sa(sa)
+	some scope in sa.scopes
+	scope == "https://www.googleapis.com/auth/cloud-platform"
+}
+
+sa_is_default if {
```

Comment:
_nit_ (not at all critical)

```suggestion
is_default_service_account if {
```

just reads nicer

Reply:
you think? the if keyword makes the name less questionable.